src/components/field/index.js

```diff
@@ -491,7 +490,8 @@ export default class TextField extends PureComponent {
     }
 
     if (this.props.mask) {
-      store.mask = this.props.mask;
+      store.mask = this.props.mask
+      store.refInput = this.inputRef
```

Comment:
This is the only non-linter change. I was getting some problems in the mask component since it was trying to access `current` on undefined, so now we'll just always pass in a ref